utils/progressbar/progressbar.go

```diff
@@ -1,29 +1,51 @@
 package progressbar
 
 import (
-	"golang.org/x/term"
 	"net/url"
 	"os"
 	"strings"
 	"sync"
 	"sync/atomic"
 	"time"
 
+	"golang.org/x/term"
+
 	"github.com/jfrog/jfrog-cli-core/v2/common/commands"
 	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
 	corelog "github.com/jfrog/jfrog-cli-core/v2/utils/log"
 	logUtils "github.com/jfrog/jfrog-cli/utils/log"
 	"github.com/jfrog/jfrog-client-go/utils"
 	ioUtils "github.com/jfrog/jfrog-client-go/utils/io"
 	"github.com/jfrog/jfrog-client-go/utils/log"
+
 	"github.com/vbauerster/mpb/v7"
 	"github.com/vbauerster/mpb/v7/decor"
 )
 
 var terminalWidth int
 
-const progressBarWidth = 20
-const progressRefreshRate = 200 * time.Millisecond
+const (
+	progressBarWidth    = 20
+	progressRefreshRate = 200 * time.Millisecond
+)
+
+// ShouldInitProgressBar func to controll whether the progress bar should be displayed.
```

Comment:
ShouldInitProgressBar func to controll whether... --> The ShouldInitProgressBar func is used to determine whether...